### PR TITLE
Fixes and tweaks to trade system

### PR DESCRIPTION
**Pull request by Zbizu**
### Pull Request Prelude
- [x] I have followed [proper The Forgotten Server code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR may be closed if the above-mentioned criteria are not fulfilled.

### Changes Proposed
update trade messages

fixes this bug:
**steps to reproduce:**
- start a trade
- partner joins a trade
- walk a few steps so there is wall a between you
- press accept, both of you

**expected:** trade cancel for both sides
**observed:** both sides see waiting for the partner to accept the trade and the window doesn't close

**Issues addressed:** this is a standalone version of trade tweaks in #3487 
compatible with both current isSightClear and the one I'm working on

[cont]: https://github.com/otland/forgottenserver/wiki/Contributing
[code]: https://github.com/otland/forgottenserver/wiki/TFS-Coding-Style-Guide

### CODE REVIEW

File: src/game.cpp [hunk 1]
```diff
@@ -2593,12 +2593,12 @@ void Game::playerRequestTrade(uint32_t playerId, const Position& pos, uint8_t st
 
 	Player* tradePartner = getPlayerByID(tradePlayerId);
 	if (!tradePartner || tradePartner == player) {
-		player->sendTextMessage(MESSAGE_INFO_DESCR, "Sorry, not possible.");
```
Comment (nekiro) [inline src/game.cpp:2596]: Why this change? It will now only show on the bottom of the screen as white text, is it correct behaviour?

Comment (Zbizu) [reply src/game.cpp:2596]: Yes. All messages except for "x wants to trade with you" come as MESSAGE_STATUS_SMALL

**Review by EPuncker — APPROVED** [review]

**Review by ranisalt — APPROVED** [review]
